dashboards-observability/public/components/application_analytics/components/application.tsx

```diff
@@ -70,14 +72,45 @@ interface AppDetailProps extends AppAnalyticsComponentDeps {
   notifications: NotificationsStart;
 }
 
-
 export function Application(props: AppDetailProps) {
-  const { pplService, dslService, timestampUtils, savedObjects, http, notifications } = props;
+  const { pplService, dslService, timestampUtils, savedObjects, http, notifications, appId, chrome, parentBreadcrumb } = props;
+  const [application, setApplication] = useState<ApplicationType>();
   const [selectedTabId, setSelectedTab] = useState<string>(TAB_OVERVIEW_ID);
   const handleContentTabClick = (selectedTab: IQueryTab) => setSelectedTab(selectedTab.id);
   const history = useHistory();
   const [toasts, setToasts] = useState<Array<Toast>>([]);
 
+  // Fetch application by id
+  const fetchAppById = async (appId: string) => {
+    return http
+      .get(`${APP_ANALYTICS_API_PREFIX}/${appId}`)
+      .then((res) => {
+        setApplication(res.application);
+      })
+      .catch((err) => {
+        setToast('Error occurred while fetching application', 'danger');
+        console.error(err);
+      })
+  }
+
+  useEffect(() => {
+    fetchAppById(appId);
+  }, [appId]);
+
+  useEffect(() => {
+    chrome.setBreadcrumbs([
+      parentBreadcrumb,
+      {
+        text: 'Application analytics',
+        href: '#/application_analytics',
+      },
+      {
+        text: application?.name,
```

Comment:
will this show up as `undefined` in UI when page is loading? if so might be better to make it blank instead of the string "undefined"

Reply:
It should not show up as anything as it's loading but I'll set it to an empty string to be safe!